build.gradle

```
@@ -1,73 +1,43 @@
 plugins {
-	id 'fabric-loom' version '1.4-SNAPSHOT'
-	id 'maven-publish'
+    id 'fabric-loom' version '1.6-SNAPSHOT'
 }
 
-sourceCompatibility = JavaVersion.VERSION_17
-targetCompatibility = JavaVersion.VERSION_17
-
-archivesBaseName = "${project.mod_id}-fabric"
 version = "${project.minecraft_version}-${project.mod_version}"
 group = project.maven_group
 
 repositories {
-	maven { url "https://maven.terraformersmc.com" }
+    maven { url "https://maven.terraformersmc.com" }
 }
 
 dependencies {
-	//to change the versions see the gradle.properties file
-	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
-	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
-
-	// Fabric API. This is technically optional, but you probably want it anyway.
-	modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
-	
-	// PSA: Some older mods, compiled on Loom 0.2.1, might have outdated Maven POMs.
-	// You may need to force-disable transitiveness on them.
-	
-	modImplementation "com.terraformersmc:modmenu:${project.mod_menu_version}"
-}
+    //to change the versions see the gradle.properties file
+    minecraft "com.mojang:minecraft:${project.minecraft_version}"
+    mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+    modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
-processResources {
-	inputs.property "version", project.version
+    // Fabric API. This is technically optional, but you probably want it anyway.
+    modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
 
-	filesMatching("fabric.mod.json") {
-		expand "version": project.version
-	}
+    modImplementation "com.terraformersmc:modmenu:${project.mod_menu_version}"
 }
```

You removed the `processResources` block even though the mod version needs to be replaced in the `fabric.mod.json` file still.
```suggestion
    // Fabric API. This is technically optional, but you probably want it anyway.
    modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"

    modImplementation "com.terraformersmc:modmenu:${project.mod_menu_version}"
}

processResources {
	inputs.property "version", project.version

	filesMatching("fabric.mod.json") {
		expand "version": project.version
	}
}
```
Not sure if my suggested change is completely right, the editor on here is kinda bad ...

---

argh, i don't know how i accidentally removed that and didn't notice it...

thank you !